Database query error Syntax error or access violation: 1064

### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### What went wrong, what did you observe?

When trying to load the poll app it starts without any polls.  After checking debug log it fails with an sql syntax error. This started after an upgrade to 7.1.1

### What did you expect, how polls should behave instead?

Polls should load fine. 

### What steps does it need to replay this bug?

1. After upgrade to 7.1.1 we cant load any.


### Installation method

Installed/updated using occ

### Installation type

Updated from previous major version (i.e. 3.x.x to 4.x.x)

### Affected polls version

7.1.1

### Which browser did you use, when experiencing the bug?

- [X] Firefox
- [X] Chrome
- [X] Chromium/Chromium based (i.e. Edge)
- [ ] Safari
- [ ] Other/Don't know

### Other browser

_No response_

### Add your browser log here

<details>
  <summary>Browser log</summary>

```json
init-collections.js:39 OCA.Sharing.ShareSearch not ready
(anonymous) @ init-collections.js:39Understand this error
votes-C_Qi3GeL.mjs:3 
        
        
       GET /apps/polls/polls?time=1718806806941 500 (Internal Server Error)
(anonymous) @ votes-C_Qi3GeL.mjs:3
xhr @ votes-C_Qi3GeL.mjs:3
Wg @ votes-C_Qi3GeL.mjs:3
_request @ votes-C_Qi3GeL.mjs:3
request @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
getPolls @ votes-C_Qi3GeL.mjs:3
list @ CalendarBlank-Cv0UBlPM.mjs:2
(anonymous) @ votes-C_Qi3GeL.mjs:3
ta.dispatch @ votes-C_Qi3GeL.mjs:3
dispatch @ votes-C_Qi3GeL.mjs:3
loadPolls @ polls-main.mjs?v=a47751d3-37:3
created @ polls-main.mjs?v=a47751d3-37:3
Fi @ votes-C_Qi3GeL.mjs:3
Ba @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
l @ votes-C_Qi3GeL.mjs:3
Lv @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
a @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
m @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
Ie @ votes-C_Qi3GeL.mjs:3
(anonymous) @ polls-main.mjs?v=a47751d3-37:3Understand this error
votes-C_Qi3GeL.mjs:3 [ERROR] polls: Error loading polls {app: 'polls', uid: 'admin', level: 3, error: Le}
log @ votes-C_Qi3GeL.mjs:3
error @ votes-C_Qi3GeL.mjs:3
list @ CalendarBlank-Cv0UBlPM.mjs:2
await in list (async)
(anonymous) @ votes-C_Qi3GeL.mjs:3
ta.dispatch @ votes-C_Qi3GeL.mjs:3
dispatch @ votes-C_Qi3GeL.mjs:3
loadPolls @ polls-main.mjs?v=a47751d3-37:3
created @ polls-main.mjs?v=a47751d3-37:3
Fi @ votes-C_Qi3GeL.mjs:3
Ba @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
l @ votes-C_Qi3GeL.mjs:3
Lv @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
a @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
m @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
Ie @ votes-C_Qi3GeL.mjs:3
(anonymous) @ polls-main.mjs?v=a47751d3-37:3Understand this error
votes-C_Qi3GeL.mjs:3 
        
        
       GET /apps/polls/administration/polls?time=1718806806942 500 (Internal Server Error)
(anonymous) @ votes-C_Qi3GeL.mjs:3
xhr @ votes-C_Qi3GeL.mjs:3
Wg @ votes-C_Qi3GeL.mjs:3
_request @ votes-C_Qi3GeL.mjs:3
request @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
getPollsForAdmin @ votes-C_Qi3GeL.mjs:3
list @ polls-main.mjs?v=a47751d3-37:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
ta.dispatch @ votes-C_Qi3GeL.mjs:3
dispatch @ votes-C_Qi3GeL.mjs:3
loadPolls @ polls-main.mjs?v=a47751d3-37:3
created @ polls-main.mjs?v=a47751d3-37:3
Fi @ votes-C_Qi3GeL.mjs:3
Ba @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
l @ votes-C_Qi3GeL.mjs:3
Lv @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
a @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
m @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
Ie @ votes-C_Qi3GeL.mjs:3
(anonymous) @ polls-main.mjs?v=a47751d3-37:3Understand this error
votes-C_Qi3GeL.mjs:3 [ERROR] polls: Error loading polls {app: 'polls', uid: 'admin', level: 3, error: Le}
log @ votes-C_Qi3GeL.mjs:3
error @ votes-C_Qi3GeL.mjs:3
list @ polls-main.mjs?v=a47751d3-37:3
await in list (async)
(anonymous) @ votes-C_Qi3GeL.mjs:3
ta.dispatch @ votes-C_Qi3GeL.mjs:3
dispatch @ votes-C_Qi3GeL.mjs:3
loadPolls @ polls-main.mjs?v=a47751d3-37:3
created @ polls-main.mjs?v=a47751d3-37:3
Fi @ votes-C_Qi3GeL.mjs:3
Ba @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
l @ votes-C_Qi3GeL.mjs:3
Lv @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
a @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
m @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
Ie @ votes-C_Qi3GeL.mjs:3
(anonymous) @ polls-main.mjs?v=a47751d3-37:3Understand this error
votes-C_Qi3GeL.mjs:3 Uncaught (in promise) Le {message: 'Request failed with status code 500', name: 'AxiosError', code: 'ERR_BAD_RESPONSE', config: {…}, request: XMLHttpRequest, …}
(anonymous) @ votes-C_Qi3GeL.mjs:3
Promise.then (async)
(anonymous) @ votes-C_Qi3GeL.mjs:3
ta.dispatch @ votes-C_Qi3GeL.mjs:3
dispatch @ votes-C_Qi3GeL.mjs:3
loadPolls @ polls-main.mjs?v=a47751d3-37:3
created @ polls-main.mjs?v=a47751d3-37:3
Fi @ votes-C_Qi3GeL.mjs:3
Ba @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
l @ votes-C_Qi3GeL.mjs:3
Lv @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
a @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
m @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
Ie @ votes-C_Qi3GeL.mjs:3
(anonymous) @ polls-main.mjs?v=a47751d3-37:3Understand this error
votes-C_Qi3GeL.mjs:3 Uncaught (in promise) Le {message: 'Request failed with status code 500', name: 'AxiosError', code: 'ERR_BAD_RESPONSE', config: {…}, request: XMLHttpRequest, …}
(anonymous) @ votes-C_Qi3GeL.mjs:3
Promise.then (async)
(anonymous) @ votes-C_Qi3GeL.mjs:3
ta.dispatch @ votes-C_Qi3GeL.mjs:3
dispatch @ votes-C_Qi3GeL.mjs:3
loadPolls @ polls-main.mjs?v=a47751d3-37:3
created @ polls-main.mjs?v=a47751d3-37:3
Fi @ votes-C_Qi3GeL.mjs:3
Ba @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
l @ votes-C_Qi3GeL.mjs:3
Lv @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
a @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
m @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
init @ votes-C_Qi3GeL.mjs:3
u @ votes-C_Qi3GeL.mjs:3
c @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
Ie @ votes-C_Qi3GeL.mjs:3
(anonymous) @ polls-main.mjs?v=a47751d3-37:3Understand this error
settings-C5h0AvBB.mjs:2 Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at ta.setPreference (settings-C5h0AvBB.mjs:2:523)
    at votes-C_Qi3GeL.mjs:3:1623460
    at votes-C_Qi3GeL.mjs:3:1619457
    at Array.forEach (<anonymous>)
    at votes-C_Qi3GeL.mjs:3:1619437
    at ta._withCommit (votes-C_Qi3GeL.mjs:3:1621252)
    at ta.commit (votes-C_Qi3GeL.mjs:3:1619412)
    at ta.commit (votes-C_Qi3GeL.mjs:3:1618961)
    at n.commit (votes-C_Qi3GeL.mjs:3:1622905)
setPreference @ settings-C5h0AvBB.mjs:2
(anonymous) @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
ta._withCommit @ votes-C_Qi3GeL.mjs:3
ta.commit @ votes-C_Qi3GeL.mjs:3
commit @ votes-C_Qi3GeL.mjs:3
n.commit @ votes-C_Qi3GeL.mjs:3
get @ settings-C5h0AvBB.mjs:2
Promise.then (async)
(anonymous) @ votes-C_Qi3GeL.mjs:3
ta.dispatch @ votes-C_Qi3GeL.mjs:3
dispatch @ votes-C_Qi3GeL.mjs:3
a.<computed> @ votes-C_Qi3GeL.mjs:3
loadContext @ polls-main.mjs?v=a47751d3-37:3
mounted @ polls-main.mjs?v=a47751d3-37:3
Fi @ votes-C_Qi3GeL.mjs:3
Ba @ votes-C_Qi3GeL.mjs:3
insert @ votes-C_Qi3GeL.mjs:3
z @ votes-C_Qi3GeL.mjs:3
(anonymous) @ votes-C_Qi3GeL.mjs:3
e._update @ votes-C_Qi3GeL.mjs:3
i @ votes-C_Qi3GeL.mjs:3
e.get @ votes-C_Qi3GeL.mjs:3
e @ votes-C_Qi3GeL.mjs:3
Nv @ votes-C_Qi3GeL.mjs:3
Ie.$mount @ votes-C_Qi3GeL.mjs:3
e._init @ votes-C_Qi3GeL.mjs:3
Ie @ votes-C_Qi3GeL.mjs:3
(anonymous) @ polls-main.mjs?v=a47751d3-37:3Understand this error
NotificationsApp.vue:470 Notifications permissions not yet requested
```

</details>

### Additional client environment information

_No response_

### NC version

Nextcloud 29

### Other Nextcloud version

_No response_

### PHP engine version

PHP 8.0

### Other PHP version

8.2.20

### Database engine

MySQL

### Database Engine version or other Database

10.6.18

### Which user-backends are you using?

- [X] Default user-backend _(database)_
- [X] LDAP/ Active Directory
- [X] SSO - SAML
- [ ] Other/Don't know

### Add your nextcloud server log here
<details>
  <summary>Server log</summary>

```json
{
  "reqId": "rhbTTZBhXWpIrOhozKv3",
  "level": 3,
  "time": "2024-06-19T14:06:15+00:00",
  "remoteAddr": "2a02:c0:4f0:b00:f816:3eff:fee6:3834",
  "user": "guno",
  "app": "index",
  "method": "GET",
  "url": "/apps/polls/polls?time=1718805976073",
  "message": "An exception occurred while executing a query: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '\",\") AS group_shares FROM `oc_polls_polls` `polls_polls` LEFT JOIN `oc_polls_...' at line 1",
  "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36",
  "version": "29.0.2.2",
  "exception": {
    "Exception": "OC\\DB\\Exceptions\\DbalException",
    "Message": "An exception occurred while executing a query: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '\",\") AS group_shares FROM `oc_polls_polls` `polls_polls` LEFT JOIN `oc_polls_...' at line 1",
    "Code": 1064,
    "Trace": [
      {
        "file": "/srv/www/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php",
        "line": 295,
        "function": "wrap",
        "class": "OC\\DB\\Exceptions\\DbalException",
        "type": "::"
      },
      {
        "file": "/srv/www/nextcloud/lib/public/AppFramework/Db/QBMapper.php",
        "line": 335,
        "function": "executeQuery",
        "class": "OC\\DB\\QueryBuilder\\QueryBuilder",
        "type": "->"
      },
      {
        "file": "/srv/www/nextcloud/apps/polls/lib/Db/PollMapper.php",
        "line": 91,
        "function": "findEntities",
        "class": "OCP\\AppFramework\\Db\\QBMapper",
        "type": "->"
      },
      {
        "file": "/srv/www/nextcloud/apps/polls/lib/Service/PollService.php",
        "line": 60,
        "function": "findForMe",
        "class": "OCA\\Polls\\Db\\PollMapper",
        "type": "->"
      },
      {
        "file": "/srv/www/nextcloud/apps/polls/lib/Controller/PollController.php",
        "line": 45,
        "function": "list",
        "class": "OCA\\Polls\\Service\\PollService",
        "type": "->"
      },
      {
        "file": "/srv/www/nextcloud/apps/polls/lib/Controller/BaseController.php",
        "line": 39,
        "function": "OCA\\Polls\\Controller\\{closure}",
        "class": "OCA\\Polls\\Controller\\PollController",
        "type": "->",
        "args": ["*** sensitive parameters replaced ***"]
      },
      {
        "file": "/srv/www/nextcloud/apps/polls/lib/Controller/PollController.php",
        "line": 42,
        "function": "response",
        "class": "OCA\\Polls\\Controller\\BaseController",
        "type": "->"
      },
      {
        "file": "/srv/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php",
        "line": 232,
        "function": "list",
        "class": "OCA\\Polls\\Controller\\PollController",
        "type": "->"
      },
      {
        "file": "/srv/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php",
        "line": 138,
        "function": "executeController",
        "class": "OC\\AppFramework\\Http\\Dispatcher",
        "type": "->"
      },
      {
        "file": "/srv/www/nextcloud/lib/private/AppFramework/App.php",
        "line": 184,
        "function": "dispatch",
        "class": "OC\\AppFramework\\Http\\Dispatcher",
        "type": "->"
      },
      {
        "file": "/srv/www/nextcloud/lib/private/Route/Router.php",
        "line": 338,
        "function": "main",
        "class": "OC\\AppFramework\\App",
        "type": "::"
      },
      {
        "file": "/srv/www/nextcloud/lib/base.php",
        "line": 1050,
        "function": "match",
        "class": "OC\\Route\\Router",
        "type": "->"
      },
      {
        "file": "/srv/www/nextcloud/index.php",
        "line": 49,
        "function": "handleRequest",
        "class": "OC",
        "type": "::"
      }
    ],
    "File": "/srv/www/nextcloud/lib/private/DB/Exceptions/DbalException.php",
    "Line": 71,
    "Previous": {
      "Exception": "Doctrine\\DBAL\\Exception\\SyntaxErrorException",
      "Message": "An exception occurred while executing a query: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '\",\") AS group_shares FROM `oc_polls_polls` `polls_polls` LEFT JOIN `oc_polls_...' at line 1",
      "Code": 1064,
      "Trace": [
        {
          "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Connection.php",
          "line": 1943,
          "function": "convert",
          "class": "Doctrine\\DBAL\\Driver\\API\\MySQL\\ExceptionConverter",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Connection.php",
          "line": 1885,
          "function": "handleDriverException",
          "class": "Doctrine\\DBAL\\Connection",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Connection.php",
          "line": 1106,
          "function": "convertExceptionDuringQuery",
          "class": "Doctrine\\DBAL\\Connection",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/lib/private/DB/Connection.php",
          "line": 328,
          "function": "executeQuery",
          "class": "Doctrine\\DBAL\\Connection",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php",
          "line": 344,
          "function": "executeQuery",
          "class": "OC\\DB\\Connection",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php",
          "line": 384,
          "function": "executeQuery",
          "class": "Doctrine\\DBAL\\Query\\QueryBuilder",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php",
          "line": 280,
          "function": "execute",
          "class": "Doctrine\\DBAL\\Query\\QueryBuilder",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php",
          "line": 293,
          "function": "execute",
          "class": "OC\\DB\\QueryBuilder\\QueryBuilder",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/lib/public/AppFramework/Db/QBMapper.php",
          "line": 335,
          "function": "executeQuery",
          "class": "OC\\DB\\QueryBuilder\\QueryBuilder",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/apps/polls/lib/Db/PollMapper.php",
          "line": 91,
          "function": "findEntities",
          "class": "OCP\\AppFramework\\Db\\QBMapper",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/apps/polls/lib/Service/PollService.php",
          "line": 60,
          "function": "findForMe",
          "class": "OCA\\Polls\\Db\\PollMapper",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/apps/polls/lib/Controller/PollController.php",
          "line": 45,
          "function": "list",
          "class": "OCA\\Polls\\Service\\PollService",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/apps/polls/lib/Controller/BaseController.php",
          "line": 39,
          "function": "OCA\\Polls\\Controller\\{closure}",
          "class": "OCA\\Polls\\Controller\\PollController",
          "type": "->",
          "args": ["*** sensitive parameters replaced ***"]
        },
        {
          "file": "/srv/www/nextcloud/apps/polls/lib/Controller/PollController.php",
          "line": 42,
          "function": "response",
          "class": "OCA\\Polls\\Controller\\BaseController",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php",
          "line": 232,
          "function": "list",
          "class": "OCA\\Polls\\Controller\\PollController",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php",
          "line": 138,
          "function": "executeController",
          "class": "OC\\AppFramework\\Http\\Dispatcher",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/lib/private/AppFramework/App.php",
          "line": 184,
          "function": "dispatch",
          "class": "OC\\AppFramework\\Http\\Dispatcher",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/lib/private/Route/Router.php",
          "line": 338,
          "function": "main",
          "class": "OC\\AppFramework\\App",
          "type": "::"
        },
        {
          "file": "/srv/www/nextcloud/lib/base.php",
          "line": 1050,
          "function": "match",
          "class": "OC\\Route\\Router",
          "type": "->"
        },
        {
          "file": "/srv/www/nextcloud/index.php",
          "line": 49,
          "function": "handleRequest",
          "class": "OC",
          "type": "::"
        }
      ],
      "File": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php",
      "Line": 86,
      "Previous": {
        "Exception": "Doctrine\\DBAL\\Driver\\PDO\\Exception",
        "Message": "SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '\",\") AS group_shares FROM `oc_polls_polls` `polls_polls` LEFT JOIN `oc_polls_...' at line 1",
        "Code": 1064,
        "Trace": [
          {
            "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php",
            "line": 132,
            "function": "new",
            "class": "Doctrine\\DBAL\\Driver\\PDO\\Exception",
            "type": "::"
          },
          {
            "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Connection.php",
            "line": 1099,
            "function": "execute",
            "class": "Doctrine\\DBAL\\Driver\\PDO\\Statement",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/lib/private/DB/Connection.php",
            "line": 328,
            "function": "executeQuery",
            "class": "Doctrine\\DBAL\\Connection",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php",
            "line": 344,
            "function": "executeQuery",
            "class": "OC\\DB\\Connection",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php",
            "line": 384,
            "function": "executeQuery",
            "class": "Doctrine\\DBAL\\Query\\QueryBuilder",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php",
            "line": 280,
            "function": "execute",
            "class": "Doctrine\\DBAL\\Query\\QueryBuilder",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php",
            "line": 293,
            "function": "execute",
            "class": "OC\\DB\\QueryBuilder\\QueryBuilder",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/lib/public/AppFramework/Db/QBMapper.php",
            "line": 335,
            "function": "executeQuery",
            "class": "OC\\DB\\QueryBuilder\\QueryBuilder",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/apps/polls/lib/Db/PollMapper.php",
            "line": 91,
            "function": "findEntities",
            "class": "OCP\\AppFramework\\Db\\QBMapper",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/apps/polls/lib/Service/PollService.php",
            "line": 60,
            "function": "findForMe",
            "class": "OCA\\Polls\\Db\\PollMapper",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/apps/polls/lib/Controller/PollController.php",
            "line": 45,
            "function": "list",
            "class": "OCA\\Polls\\Service\\PollService",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/apps/polls/lib/Controller/BaseController.php",
            "line": 39,
            "function": "OCA\\Polls\\Controller\\{closure}",
            "class": "OCA\\Polls\\Controller\\PollController",
            "type": "->",
            "args": ["*** sensitive parameters replaced ***"]
          },
          {
            "file": "/srv/www/nextcloud/apps/polls/lib/Controller/PollController.php",
            "line": 42,
            "function": "response",
            "class": "OCA\\Polls\\Controller\\BaseController",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php",
            "line": 232,
            "function": "list",
            "class": "OCA\\Polls\\Controller\\PollController",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php",
            "line": 138,
            "function": "executeController",
            "class": "OC\\AppFramework\\Http\\Dispatcher",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/lib/private/AppFramework/App.php",
            "line": 184,
            "function": "dispatch",
            "class": "OC\\AppFramework\\Http\\Dispatcher",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/lib/private/Route/Router.php",
            "line": 338,
            "function": "main",
            "class": "OC\\AppFramework\\App",
            "type": "::"
          },
          {
            "file": "/srv/www/nextcloud/lib/base.php",
            "line": 1050,
            "function": "match",
            "class": "OC\\Route\\Router",
            "type": "->"
          },
          {
            "file": "/srv/www/nextcloud/index.php",
            "line": 49,
            "function": "handleRequest",
            "class": "OC",
            "type": "::"
          }
        ],
        "File": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Driver/PDO/Exception.php",
        "Line": 28,
        "Previous": {
          "Exception": "PDOException",
          "Message": "SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '\",\") AS group_shares FROM `oc_polls_polls` `polls_polls` LEFT JOIN `oc_polls_...' at line 1",
          "Code": "42000",
          "Trace": [
            {
              "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php",
              "line": 130,
              "function": "execute",
              "class": "PDOStatement",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Connection.php",
              "line": 1099,
              "function": "execute",
              "class": "Doctrine\\DBAL\\Driver\\PDO\\Statement",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/lib/private/DB/Connection.php",
              "line": 328,
              "function": "executeQuery",
              "class": "Doctrine\\DBAL\\Connection",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php",
              "line": 344,
              "function": "executeQuery",
              "class": "OC\\DB\\Connection",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php",
              "line": 384,
              "function": "executeQuery",
              "class": "Doctrine\\DBAL\\Query\\QueryBuilder",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php",
              "line": 280,
              "function": "execute",
              "class": "Doctrine\\DBAL\\Query\\QueryBuilder",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php",
              "line": 293,
              "function": "execute",
              "class": "OC\\DB\\QueryBuilder\\QueryBuilder",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/lib/public/AppFramework/Db/QBMapper.php",
              "line": 335,
              "function": "executeQuery",
              "class": "OC\\DB\\QueryBuilder\\QueryBuilder",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/apps/polls/lib/Db/PollMapper.php",
              "line": 91,
              "function": "findEntities",
              "class": "OCP\\AppFramework\\Db\\QBMapper",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/apps/polls/lib/Service/PollService.php",
              "line": 60,
              "function": "findForMe",
              "class": "OCA\\Polls\\Db\\PollMapper",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/apps/polls/lib/Controller/PollController.php",
              "line": 45,
              "function": "list",
              "class": "OCA\\Polls\\Service\\PollService",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/apps/polls/lib/Controller/BaseController.php",
              "line": 39,
              "function": "OCA\\Polls\\Controller\\{closure}",
              "class": "OCA\\Polls\\Controller\\PollController",
              "type": "->",
              "args": ["*** sensitive parameters replaced ***"]
            },
            {
              "file": "/srv/www/nextcloud/apps/polls/lib/Controller/PollController.php",
              "line": 42,
              "function": "response",
              "class": "OCA\\Polls\\Controller\\BaseController",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php",
              "line": 232,
              "function": "list",
              "class": "OCA\\Polls\\Controller\\PollController",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php",
              "line": 138,
              "function": "executeController",
              "class": "OC\\AppFramework\\Http\\Dispatcher",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/lib/private/AppFramework/App.php",
              "line": 184,
              "function": "dispatch",
              "class": "OC\\AppFramework\\Http\\Dispatcher",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/lib/private/Route/Router.php",
              "line": 338,
              "function": "main",
              "class": "OC\\AppFramework\\App",
              "type": "::"
            },
            {
              "file": "/srv/www/nextcloud/lib/base.php",
              "line": 1050,
              "function": "match",
              "class": "OC\\Route\\Router",
              "type": "->"
            },
            {
              "file": "/srv/www/nextcloud/index.php",
              "line": 49,
              "function": "handleRequest",
              "class": "OC",
              "type": "::"
            }
          ],
          "File": "/srv/www/nextcloud/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php",
          "Line": 130
        }
      }
    },
    "message": "An exception occurred while executing a query: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '\",\") AS group_shares FROM `oc_polls_polls` `polls_polls` LEFT JOIN `oc_polls_...' at line 1",
    "exception": {},
    "CustomMessage": "An exception occurred while executing a query: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '\",\") AS group_shares FROM `oc_polls_polls` `polls_polls` LEFT JOIN `oc_polls_...' at line 1"
  }
}
```

</details>


### Additional environment informations

_No response_

### Configuration report

```json
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            ""
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "skeletondirectory": "",
        "templatedirectory": "",
        "dbtype": "mysql",
        "version": "29.0.2.2",
        "overwrite.cli.url": "https:\/\/",
        "htaccess.RewriteBase": "\/",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "filelocking.enabled": true,
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0.5,
            "dbindex": 0
        },
        "default_phone_region": "SE",
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "loglevel": 3,
        "maintenance": false,
        "maintenance_window_start": 1,
        "mail_smtpmode": "smtp",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "tls",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "allow_user_to_change_display_name": "false",
        "lost_password_link": "disabled",
        "simpleSignUpLink.shown": false,
        "auth.webauthn.enabled": false,
        "theme": "",
        "updater.release.channel": "stable",
        "user_oidc": {
            "auto_provision": false
        },
        "ldapProviderFactory": "OCA\\User_LDAP\\LDAPProviderFactory",
        "mail_sendmailmode": "smtp",
        "log.condition": {
            "apps": [
                "admin_audit"
            ]
        },
        "preview_concurrency_new": "8",
        "preview_concurrency_all": "16",
        "enabledPreviewProviders": [
            "OC\\Preview\\Imaginary",
            "OC\\Preview\\Image",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\OpenDocument",
            "OC\\Preview\\Movie",
            "OC\\Preview\\Krita"
        ],
        "preview_imaginary_url": "***REMOVED SENSITIVE VALUE***",
        "preview_max_x": "2048",
        "preview_max_y": "2048",
        "jpeg_quality": "60",
        "preview_max_memory": "256",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***"
    }
}
```


### List of activated Apps

```shell
- activity: 2.21.1
  - admin_audit: 1.19.0
  - calendar: 4.7.6
  - circles: 29.0.0-dev
  - cloud_federation_api: 1.12.0
  - collectives: 2.11.0
  - comments: 1.19.0
  - contacts: 6.0.0
  - contactsinteraction: 1.10.0
  - dashboard: 7.9.0
  - dav: 1.30.1
  - deck: 1.13.0
  - drawio: 3.0.2
  - federatedfilesharing: 1.19.0
  - federation: 1.19.0
  - files: 2.1.0
  - files_antivirus: 5.5.4
  - files_downloadlimit: 2.0.0
  - files_fulltextsearch: 29.0.0
  - files_lock: 29.0.1
  - files_pdfviewer: 2.10.0
  - files_reminders: 1.2.0
  - files_scripts: 4.1.0
  - files_sharing: 1.21.0
  - files_trashbin: 1.19.0
  - files_versions: 1.22.0
  - firstrunwizard: 2.18.0
  - flow_notifications: 1.9.0
  - forms: 4.2.4
  - fulltextsearch: 29.0.0
  - fulltextsearch_elasticsearch: 29.0.1
  - groupfolders: 17.0.1
  - impersonate: 1.16.0
  - integration_gitlab: 1.0.19
  - integration_mattermost: 1.0.7
  - integration_zimbra: 1.0.9
  - logreader: 2.14.0
  - lookup_server_connector: 1.17.0
  - nextcloud_announcements: 1.18.0
  - notes: 4.10.0
  - notifications: 2.17.0
  - notify_push: 0.6.12
  - oauth2: 1.17.0
  - password_policy: 1.19.0
  - photos: 2.5.0
  - polls: 7.1.0
  - privacy: 1.13.0
  - provisioning_api: 1.19.0
  - recommendations: 2.1.0
  - related_resources: 1.4.0
  - richdocuments: 8.4.2
  - serverinfo: 1.19.0
  - settings: 1.12.0
  - sharebymail: 1.19.0
  - spreed: 19.0.1
  - support: 1.12.0
  - survey_client: 1.17.0
  - suspicious_login: 7.0.0
  - systemtags: 1.19.0
  - tables: 0.7.3
  - text: 3.10.0
  - theming: 2.4.0
  - twofactor_backupcodes: 1.18.0
  - updatenotification: 1.19.1
  - user_ldap: 1.20.0
  - user_oidc: 5.0.2
  - user_status: 1.9.0
  - viewer: 2.3.0
  - weather_status: 1.9.0
  - welcome: 1.1.0
  - workflowengine: 2.11.0
```


### Nextcloud Signing status

```shell
No errors have been found.
```


### Additional Information

_No response_

--- a/lib/Db/PollMapper.php
+++ b/lib/Db/PollMapper.php
@@ -243,10 +243,10 @@ class PollMapper extends QBMapper {
 			$qb->addSelect($qb->createFunction('group_concat(replace(distinct ' . $joinAlias . '.user_id ,\'\',\'\'), \''. self::CONCAT_SEPARATOR . '\') AS group_shares'));
 
 		} elseif ($this->db->getDatabasePlatform() instanceof MySQLPlatform) {
-			$qb->addSelect($qb->createFunction('group_concat(distinct ' . $joinAlias . '.user_id SEPARATOR "'. self::CONCAT_SEPARATOR . '") AS group_shares'));
+			$qb->addSelect($qb->createFunction('group_concat(distinct ' . $joinAlias . '.user_id SEPARATOR \''. self::CONCAT_SEPARATOR . '\') AS group_shares'));
 
 		} else {
-			$qb->addSelect($qb->createFunction('group_concat(distinct ' . $joinAlias . '.user_id SEPARATOR "'. self::CONCAT_SEPARATOR . '") AS group_shares'));
+			$qb->addSelect($qb->createFunction('group_concat(distinct ' . $joinAlias . '.user_id SEPARATOR \''. self::CONCAT_SEPARATOR . '\') AS group_shares'));
 		}
 
 		$qb->leftJoin(

